src/Hubspot/SyncToService.php

```diff
@@ -69,8 +83,15 @@ public function __construct(
    *   The entity we're syncing (ie. user/order/product variation).
    */
   public function sync(EntityInterface $entity) {
-    // TODO: Depending on which entity we're trying to sync, trigger the
-    // appropriate action.
+    // Dispatch an event to allow modules to tell us which Hubspot entity and ID
+    // to sync this Drupal entity with.
+    $entity_mapping = [];
+    $event = new EntityMappingEvent($entity_mapping);
+    $this->eventDispatcher->dispatch(EntityMappingEvent::EVENT_NAME, $event);
+
+    if (empty($entity_mapping)) {
```

Comment:
Since the ID won't be available if the entity isn't synced with Hubspot yet, we don't need need to check if the ID exists.